multihost_test/Bugzillas/test_Regression.py

```diff
@@ -464,3 +471,63 @@ def test_bz450262(self, multihost, create_backup):
         client.run_command(f"rm -f {output_file}")
         client.run_command(f"groupdel {second_group}")
         client.run_command(f"groupdel {first_group}")
+
+    @pytest.mark.tier1
+    def test_bz1285547(self, multihost, create_backup):
+        """
+        :title:pam_lastlog unable to reset locked account
+        :id: 94626b58-f7c6-11ee-bf95-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1285547
+        :steps:
+          1. Creates a new user test1.
+          2. Clear the contents of the lastlog file and sets its context using restorecon.
+          3. Generates options for the lastlog command and checks if they are all present.
+          4. Simulates a login for user test1 that is five days in the past using log_back_in_time.
+          5. Checks the last login entry for test1 to ensure it's not showing "Never logged in".
+          6. Tests various options of lastlog command (--before, --time).
+          7. Clears the last login entry for test1 and checks if it shows "Never logged in" again.
+          8. Sets a new last login time for test1.
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+          4. Should succeed
+          5. Should succeed
+          6. Should succeed
+          7. Should succeed
+          8. Should succeed
+        """
+        client = multihost.client[0]
+        log = "/var/log/lastlog"
+        options = "/tmp/options.txt"
+        user = "test1"
+        opt = ["before", "help", "root", "time", "user", "set", "clear"]
+        client.run_command("useradd test1")
+        client.run_command(f"> {log}")
+        client.run_command(f"restorecon -vF {log}")
+        client.run_command(f"lastlog --help | grep '^[[:blank:]]*-[[:alpha:]],' > {options}")
+        real_data = client.run_command(f"cat {options}").stdout_text
+        for data in opt:
+            assert data in real_data
+        assert int(client.run_command(f"cat {options}  | wc -l").stdout_text.split()[0]) == len(opt)
```

Comment:
I think this should go in a separate test case.

Reply:
This contains automation for two bzs